src/modules/sharepoint/list-item-filter/field-equals.list-item-filter.ts

```diff
@@ -0,0 +1,13 @@
+import { ListItemFilter } from './list-item-filter.interface';
```

Comment:
Looking at the file name -- are `fields` only on `list-item` or can this folder be renamed?

Reply:
Yes, fields are a listItem thing rather than a general Microsoft Graph thing :)